orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/FindImageFromClusterTask.groovy

```diff
@@ -124,7 +125,8 @@ class FindImageFromClusterTask extends AbstractCloudProviderAwareTask implements
     Map<Location, String> imageIds = [:]
     Set<String> inferredRegions = new HashSet<>()
 
-    if (cloudProvider == 'aws') {
+    Boolean skipRegionDetection = Boolean.TRUE == config.skipRegionDetection
+    if (cloudProvider == 'aws' && !skipRegionDetection) {
```

Comment:
any particular reason not to just use `!config.skipRegionDetection` here, I think null and false will evaluate as expected

Reply:
Nah, none at all. I sadly just lifted that from the bake stage verbatim. Changed.